ofnetMaster.go

```diff
@@ -126,6 +126,8 @@ func (self *OfnetMaster) RegisterNode(hostInfo *OfnetNode, ret *bool) error {
 
 // Add an Endpoint
 func (self *OfnetMaster) EndpointAdd(ep *OfnetEndpoint, ret *bool) error {
+
+	log.Infof("Received Endpoint CReate from Remote netplugin")
```

Comment:
Can we remove this or move it to log.Debugf() since its for debug purposes only?